src/graphics.rs

```diff
@@ -284,6 +277,8 @@ impl RenderPass {
     }
 }
 
+/// A type that represents the different types of actions one can apply
+/// to a frame buffer during Render passes
```

Comment:
- at the start of a render pass, not during. Maybe note that it's mostly stuff to reinitialize state by doing things like clearing buffers to specific colors/depths/whatnot?